xml/Microsoft.Extensions.Logging.Abstractions/NullLogger.xml

```diff
@@ -59,8 +59,8 @@
       <Docs>
         <typeparam name="TState">To be added.</typeparam>
         <param name="state">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Begins a logical operation scope.</summary>
+        <returns>An <see cref="T:System.IDisposable" /> that ends the logical operation scope on dispose.</returns>
```

Comment:
Usually natural language is preferred for the descriptions, since the data type is already going to show up next to in in MS Docs:
```suggestion
        <returns>A disposable object that eends the logical operation scope on dispose.</returns>
```